dhis-2/dhis-services/dhis-service-administration/src/main/java/org/hisp/dhis/dataintegrity/DataIntegrityYamlReader.java

```diff
@@ -147,8 +147,7 @@ private static String sanitiseSQL( String sql )
             .replaceAll( "select '[^']+' as [^,]+,", "select " )
             .replaceAll( "'[^']+' as description", "" )
             .replace( "::varchar", "" )
-            .replace( "|| '%'", "" )
-            .replaceAll( " as [^,]+,", "," ) );
```

Comment:
FYI: This was placed to avoid issues with SQL when used with H2 but it was way to aggressive and cut out sections from SQL that were required when used with postgres.